lib/librato/rails/subscribers/job.rb

```diff
@@ -9,11 +9,17 @@ module Subscribers
         ActiveSupport::Notifications.subscribe "#{metric}.active_job" do |*args|
 
           event = ActiveSupport::Notifications::Event.new(*args)
+
           tags = {
-            adapter: event.payload[:adapter].to_s.demodulize.underscore,
+            adapter: event.payload[:adapter].class.to_s.demodulize.underscore,
             job: event.payload[:job].class.to_s.demodulize.underscore
           }
 
+          Librato::Rails::VersionSpecifier.supported(max: '4.2') do
```

Comment:
Since this is already inside the `Librato::Rails` namespace, this can just be `VersionSpecifier` unless you are intentionally being more explicit.